pkg/agent/openflow/client.go

```diff
@@ -1150,3 +1144,15 @@ func (c *client) SendUDPPacketOut(
 	packetOutObj := packetOutBuilder.Done()
 	return c.bridge.SendPacketOut(packetOutObj)
 }
+
+func addActionsToPacketOut(packetOutBuilder binding.PacketOutBuilder, packetOutType binding.PacketOutType) binding.PacketOutBuilder {
+	switch packetOutType {
+	case binding.RejectServiceLocal:
+		tableID := uint8(conntrackTable)
+		packetOutBuilder = packetOutBuilder.AddResubmitAction(nil, &tableID)
+	case binding.RejectLocalToRemote:
+		tableID := uint8(l3ForwardingTable)
+		packetOutBuilder = packetOutBuilder.AddResubmitAction(nil, &tableID)
+	}
```

Comment:
Did you miss case "RejectServiceRemoteToLocal" here?

Reply:
No. "RejectServiceRemoteToLocal" case doesn't need additional `Actions`. Because this case is triggered when Service traffic that has remote Endpoints is rejected by the egress rule. srcMAC, dstMAC, srcIP and dstIP are all as we expected. And the original packet has committed CT. This reject response could be handled correctly by our pipeline.